documentation/migration-guide-2.0.0.md

```diff
@@ -0,0 +1,171 @@
+# Uber iOS SDK 2.0.0 - Migration Guide
+The Uber Rides SDK has been revamped and renamed to the Uber iOS SDK. Among other things, this new release includes a simplified API, better security for mobile OAuth, and updated platform support.
+
+There are a few breaking changes that must be handled when updating the SDK. This guide will help you navigate the major code changes necessary to make the upgrade.
+
+## New + Updated Features
+### Swift Package Manager Support
+Swift Package Manager is now the preferred way to integrate the Uber iOS SDK. Support for other third party package managers has been dropped for this release.
```

Comment:
What's the reasoning for this?

Reply:
Mobile Foundations is strongly encouraging this change. If the impact is too large we may revisit this decision down the line, but for now their stance is that SPM should be the only option going forward.